solution.txt

```diff
@@ -142,22 +136,11 @@ Exercise 12:
 */
 
 // Complete Exercise 12 below...
-
 let fizz = [];
 let buzz = [];
 let fizzbuzz = [];
+nums.forEach(element => { if(element % 5 === 0 && element % 3 === 0){ fizzbuzz.push(element)} else if(element % 5 === 0){buzz.push(element)} else if(element % 3 === 0){fizz.push(element)}});
```

Comment:
this looks good to me, but please use indentation! indentation is super important as far as making your code readable, and readable code is code that others will want to collaborate with you on / hire you for.